### PR TITLE
Release changelog for 2.35

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,128 @@
...
+
+  [ Aleix Pol ]
+  * repo: return a proper value in DummyRepo
+  * common: do not fail over on empty or faulty lines in os-release
+
+  [ Jeff Dickey ]
+  * ci: install git in Dockerfile for '{version: git}' usage (#1575)
+
+  [ Michael Vogt ]
+  * meta: add and adapter property for apps
```
Comment (kyrofa) [inline debian/changelog]: There are a few lines that don't have PR numbers. Were those merged differently?

**Comment by sergiusens** [comment]
On mié, nov 1, 2017 at 5:23 PM, Kyle Fazzari 
<notifications@github.com> wrote:
> @kyrofa commented on this pull request.
> 
> In debian/changelog:
> 
> > +  * lxd: split container classes into different files (#1627)
> +  * cli: update parts cache in the container (#1546)
> +  * lifecycle: clean after deleting container (#1587)
> +
> +  [ James Beedy ]
> +  * ruby plugin: new plugin
> +
> +  [ Aleix Pol ]
> +  * repo: return a proper value in DummyRepo
> +  * common: do not fail over on empty or faulty lines in os-release
> +
> +  [ Jeff Dickey ]
> +  * ci: install git in Dockerfile for '{version: git}' usage (#1575)
> +
> +  [ Michael Vogt ]
> +  * meta: add and adapter property for apps
> There are a few lines that don't have PR numbers. Were those merged 
> differently?
> 

Those were the rebased ones before we decided to squash and merge again.
>

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,128 @@
...
+  * tests: don't hit internet in ros2 units (#1619)
+  * states: add scriptlets to build state (#1618)
+  * schema: sync patterns with snapd (#1622)
+  * snap: remove leaking LD_LIBRARY_PATH (#1635)
+  * store: guide to account creation upon login (#1616)
+  * repo: add elementary to deb distros (#1637)
+
+  [ Leo Arias ]
+  * typo: replace occured with occurred
+  * python node: record installed node packages in manifest
```
Comment (elopio) [inline debian/changelog]: here's an error, should be node plugin instead of python node

Comment (sergiusens) [reply debian/changelog]: fixed

**Comment by sergiusens** [comment]
We will wait for #1636 and move forward after that makes it in with the adt testing across the board.

**Comment by kyrofa** [comment]
#1636 has been merged.

**Comment by elopio** [comment]
I checked a few of the errors.

For bionic to work, I enabled it on the daily PPA. I'm not sure why we are still using that PPA on these unit tests, or where is that configured :/
For fixing the ros errors introduced on the latest branch, I proposed #1651. An easier alternative would be to add pyramid to debian/tests/control.
About the open socket warning, I'm too tired now to keep debugging. I reported https://github.com/msabramo/requests-unixsocket/issues/36

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,161 @@
...
+
+  [ Martin Wimpress ]
+  * nodejs plugin: update default node engine to 6.11.4 (#1589)
+
+  [ Colin Watson ]
+  * options: fix core-dynamic-linker on ppc64el/s390x (#1600)
+
+  [ Paolo Pisati ]
+  * kbuild plugin: if the parts build dir already contains a .config file, return immediately (#1606)
+  * cross compilation: enable cross compilation of i386 kernel on x86-64 … (#1613)
```
Comment (kyrofa) [inline debian/changelog]: The ellipsis looks a bit odd here. Can we fill it out, or is it too long?

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,161 @@
...
+  [ Alfonso Sanchez Beato ]
+  * kernel plugin: use latest stable core snap (#1624)
+
+  [ Nathan Haines ]
+  * ruby plugin: new stable release 2.4.2 (#1645)
+
+  [ Marius Gripsgard ]
+  * sources: workaround for ZipFile.extractall not preserving permissions (#1723)
+
+ -- Sergio Schvezov <sergio.schvezov@canonical.com>  Wed, 01 Nov 2017 19:41:33 +0000
```
Comment (kyrofa) [inline debian/changelog]: Worth updating this?

**Comment by kyrofa** [comment]
Note that LP: [#1732076](https://bugs.launchpad.net/snapcraft/+bug/1732076) will likely get in the way of this deb being released. It just started surfacing today in adt.

File: debian/changelog (hunk 4 above)
Comment (sergiusens) [reply debian/changelog]: wow, let's leave this as a reminder of how long it took!

**Comment by kyrofa** [comment]
\o/

**Pull request merged**